src/Hyperwallet/Hyperwallet.php

```diff
@@ -1803,4 +1805,57 @@ private function updateTransferMethod($userToken, $transferMethod, $transferMeth
                 ), $transferMethod, array());
     }
 
+    /**
+     * Create a bank card status transition
+     *
+     * @param string $userToken The user token
+     * @param UserStatusTransition $transition The status transition
+     * @return UserStatusTransition
+     *
+     * @throws HyperwalletArgumentException
+     * @throws HyperwalletApiException
+     */
+    public function updateVerificationStatus($userToken, $verificationStatus){
+        if (empty($userToken)) {
+            throw new HyperwalletArgumentException('userToken is required!');
+        }
+        if (empty($verificationStatus)) {
+            throw new HyperwalletArgumentException('verification status is required!');
+        }
+        if($verificationStatus != User::VERIFICATION_STATUS_REQUESTED){
+            throw new HyperwalletArgumentException("Expected verification status is REQUESTED!");
+        }
+        $user = $this->getUser($userToken);
+        $fromValidationStatus = $user->getVerificationStatus();
+        $expectedFromValues = array(User::VERIFICATION_STATUS_REQUIRED,User::VERIFICATION_STATUS_NOT_REQUIRED);
+        if(!in_array($fromValidationStatus,$expectedFromValues)){
+            throw new HyperwalletException("The from verification status is expected to be 'Required' or 'Not required'");
+        }
```

Comment:
Not sure we need to handle these validations in the SDK?

Reply:
This is the interface level validations to stop the request all the way to the remote server. If this is not provided, the user has to handle the received exception in the form of http response as well.

Reply:
Even I think this has to be handled in the hw2 layer and not in sdk. Because to know whether the current verification status is REQUIRED or NOT_REQUIRED, we are making a getUser call.